### PR TITLE
QoS test params for J2C+ based HWSKUs

### PR DESCRIPTION
**Pull request by vmittal-msft**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
To add QoS test params for J2C+ HWSKUs

#### How did you do it?
Updated qos params in qos.yml file.

#### How did you verify/test it?
Verification still in progress on real HWSKUs. These are starting values.

### CODE REVIEW

**Comment by vmittal-msft** [comment]
@ sanmalho-git Can you review and approve ?

**Comment by sanmalho-git** [comment]
@vmittal-msft - I don't have permissions to approve.  Also, don't have jr2 box to validate the changes.

File: tests/qos/files/qos.yml [hunk 1]
```diff
@@ -2364,3 +2364,464 @@ qos_params:
...
+                q3_num_of_pkts: 300
+                q4_num_of_pkts: 300
+                q5_num_of_pkts: 80
+                q6_num_of_pkts: 80
+                limit: 80
+                lossy_weight: 8
+                lossless_weight: 30
+            hdrm_pool_wm_multiplier: 1
+            cell_size: 4096
+    j2c+:
```
Comment (arlakshm) [inline tests/qos/files/qos.yml:2521]: Do we code to identify if LC is `jr2` or `j2c+` ?

Comment (vmittal-msft) [reply tests/qos/files/qos.yml:2521]: image

We specify that in test infra.

**Review by arlakshm — APPROVED** [review]

**Pull request merged**